### PR TITLE
file

### PR DESCRIPTION
**Pull request by Harshal662**
Closes: #29

### CODE REVIEW

**Comment by tungbq** [comment]
@Harshal662 could you update the PR title to have more specific meaning? Thanks (May be we could refer to the same title of the task #29 that we are working on)
Same as: https://github.com/tungbq/devops-basic/pull/45#issuecomment-1575572422

File: shell/examples/README.md [hunk 1]
```diff
@@ -1 +1,7 @@
 # Collection of hands on shell example
+
+In this script, we first define the output file name as file_list.txt. Then, we use the ls command with the *.txt pattern to list all .txt files in the current directory. The > operator redirects the output of the ls command to the specified output file.
+
+Finally, we display a message indicating the completion of the task, specifying the name of the output file.
+
+ After running the script, you will find the list of .txt files in the current directory stored in the file_list.txt file.
```
Comment (tungbq) [inline shell/examples/README.md:7]: It's good, but I think we should move these lines to `shell/examples/list.sh` as a comment

Comment (Harshal662) [reply shell/examples/README.md:7]: okay. i will added these. should i delete these lines from readme file?

Comment (tungbq) [reply shell/examples/README.md:7]: Yes, please remove them

**Comment by Harshal662** [comment]
made changes

**Review by tungbq — APPROVED** [review]
Thanks!

**Pull request merged**